coralproject/talk
# Fix Python install error during Docker builds

## What does this PR do?

Specify python3 in Dockerfile to fix our image builds.

node:12-alpine maintainers have "helpfully" removed the default python -> python3 alias for the apk registry. This breaks python installs in our Docker builds.

Specify python3 so it stops breaking and our builds can be happy again.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Run the following Docker build command:

`docker build -t test-build-coral -f Dockerfile .`
(this is just building Coral with a dummy name of `test-build-coral` so you can easily delete the test image afterwards)

See that the build makes it past the `RUN apk --no-cache add git python3` step specified in the `Dockerfile`.

If you feel like it, try setting the above mentioned step back to the following to see that the old way of installing pythond id not work.

`RUN apk --no-cache add git python` <-- this will error that it can't find Python
 
## How do we deploy this PR?

Does not affect deploy procedure.

## Credits

Thanks to @wyattjoh for suggesting to try installing a specific python (i.e. `python3`) that resulted in this fix. 😄
